.coderabbit.yaml

```
@@ -0,0 +1,73 @@
+    paths: ["**/*.test.*", "**/tests/**"]  # تمرکز روی فایل‌های تست اما پیشنهاد برای دیگران
+
+disable_rules: []  # هیچ قانونی را غیرفعال نکنید؛ همه را برای سختگیری کامل فعال نگه دارید
+
+# تنظیمات چت‌ها
+chats:
+  auto_reply: false  # غیرفعال کردن پاسخ‌های خودکار برای تشویق تعامل دستی و بحث‌های عمیق‌تر
+  require_tests: true  # الزام تست‌ها برای تغییرات در چت‌ها
+  require_tasks: true  # الزام شکستن تغییرات به وظایف
+  require_security: true  # همیشه بررسی امنیت در پاسخ‌های چت
+
+# تنظیمات ابزارها 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)